### PR TITLE
enhance: unify data generation test APIs

### PR DESCRIPTION
**Pull request by cydrain**
Issue: #22837

### CODE REVIEW

File: tests/integration/util_insert.go [hunk 1]
```diff
@@ -120,97 +114,27 @@ func NewVarCharSameFieldData(fieldName string, numRows int, value string) *schem
 }
 
 func NewStringFieldData(fieldName string, numRows int) *schemapb.FieldData {
```
Comment (bigsheeper) [inline tests/integration/util_insert.go:116]: Just remove this function?

Comment (cydrain) [reply tests/integration/util_insert.go:116]: direct removal will cause too many code change

**Pull request merged**